.planning/phases/05-test-coverage/05-01-PLAN.md

```
@@ -0,0 +1,221 @@
+  key_links:
+    - from: "ReviewTests.cs"
+      to: "CoffeePeek.Shops.Domain/Aggregates/ReviewAggregate/ReviewAction.cs"
+      via: "Review.Create вызов с вариацией параметров"
+      pattern: "Review\\.Create"
+    - from: "RatingTests.cs"
+      to: "CoffeePeek.Shops.Domain/Entities/Rating.cs"
+      via: "Rating.Create вызов с boundary values"
+      pattern: "Rating\\.Create"
+---
+
+<objective>
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)